### DISCUSSION TITLE
zh/docs/gameshell-ext/screen

### DISCUSSION THREAD

**Discussion by giscus** [ClockworkPiFans/manual#1] [Announcements]
# zh/docs/gameshell-ext/screen

效果展示 改装须知 改装大屏幕后，GameShell 最终的体积会变大，重量会略微增加，如果介意，请不要考虑改装。 材料准备 选购清单: 屏幕（二选一）; 选项1: 3.5

https://manual.clockworkpi-fans.com/zh/docs/gameshell-ext/screen.html

**Comment by zzxzzk115** [comment]
### 工作繁忙，诚招伙伴
最近工作比较繁忙，希望感兴趣的小伙伴，可以加入到这个组织中，我们一起建设网站和手册相关内容。
